docs/data/base/getting-started/quickstart/quickstart.md

```diff
@@ -0,0 +1,235 @@
+# Quickstart
+
+<p class="description">Get started with Base UI, a library of headless ("unstyled") React UI components and low-level hooks.</p>
+
+## Installation
+
+`@mui/base` is completely standalone – run one of the following commands to add Base UI to your React project:
+
+### With npm
+
+```sh
+npm install @mui/base
+```
+
+### With yarn
+
+```sh
+yarn add @mui/base
+```
+
+### With pnpm
+
+```sh
+pnpm add @mui/base
+```
+
+### Peer dependencies
+
+<!-- #react-peer-version -->
+
+[`react`](https://www.npmjs.com/package/react) >= 17.0.0 and [`react-dom`](https://www.npmjs.com/package/react-dom) >= 17.0.0 are peer dependencies.
+
+## Implementing a Button
+
+This is a quick tutorial that goes through the basics of using and styling Base UI components by replicating a button from GitHub's UI, using their [Primer design system](https://primer.style) as a reference.
+Base UI components help you abstract away low-level UI implementation details such as accessibility patterns, cross-browser compatibility, event handling, and more, while remaining unopinionated about styling, giving you complete control over your app's CSS.
+
+{{"demo": "Tutorial.js", "defaultCodeOpen": false, "hideToolbar": true}}
+
+### Components and hooks
+
+Base UI provides a Button component and a `useButton` hook.
```

Comment:
```suggestion
Base UI provides a Button component and a useButton hook.
```

or "a `Button` component and a `useButton` hook"

Reply:
@michaldudak according to the [Style Guide](https://www.notion.so/mui-org/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b), the way it's written currently is correct. (Of course it can always be changed as we see fit.)

If you wanted to put both in backticks, the preferred style would be `<Button />` and `useButton`. If you think that neither of them should have backticks, then there are many other instances of `useSomeHook` throughout the Base and Joy docs that we'll want to update to remove backticks.

Reply:
Let's go with `<Button />` here as it's the first occurrence of the button component, subsequent ones will have no backtick